packages/headless/src/features/search/legacy/search-actions.ts

```diff
@@ -20,7 +20,7 @@ import {
   updateInstantResultsQuery,
 } from '../../instant-results/instant-results-actions.js';
 import {buildSearchAndFoldingLoadCollectionRequest} from '../../search-and-folding/legacy/search-and-folding-request.js';
-import {updateSearchAction} from '../search-actions.js';
+import {setReadyToGenerateAnswerAction} from '../search-actions.js';
```

Comment:
This cannot be the solution,

This creates a big coupling problem between the search actions and the generate answer action.

We cannot have the search action be impacted or responsible for timing issues related to the generated answer. It is fundamentally incorrect.